### ISSUE TITLE
Unlock dialog should have archive listed

### ISSUE THREAD

**Issue by perry-mitchell** [buttercup/buttercup-desktop#554]
When unlocking an archive (which can be triggered in a variety of ways) it'd be great to have the name (and maybe type?) of the archive in the unlock window.

**Comment by sallar** [comment]
@perry-mitchell adding password strength meter #531 requires re-designing this unlock dialog. Perhaps they can be done together with this

**Comment by perry-mitchell** [comment]
Sure.. let’s take this after the other work perhaps? 😁

**Comment by sallar** [comment]
@perry-mitchell yes, I just added all three issues to a milestone which can be bundled together in the future.

**Comment by julianpoemp** [comment]
+1

It would be very useful to have the name of the vault in the unlock window. If you have more than one vaults you don't know which vault is currently selected for unlock.
